controlnext/views.py

```diff
@@ -67,17 +55,29 @@ def post(self, request, random_url_slug):
         return RestResponse(request.POST)
 
 
+class Http403View(TemplateView):
```

Comment:
Kan je een TemplateView ook makkelijk een response code (403, in dit geval) terug laten geven? Da's wel zo netjes.